README.md

```diff
@@ -93,14 +93,14 @@ $ sudo xattr -rd com.apple.quarantine /Applications/GridPlayer.app
 $ pip install -U gridplayer
 ```
 
-**Python 3.8 or later required.**
+**Python 3.8 or later required.** (_note: 3.12 and possibly a few previous versions do not work_)
 
 This type of installation will also require a `vlc` package present in your system.
```

Comment:
I don't understand what a "`vlc` package` means 😓 
Do you mean install VLC into `/Program Files` ?